### PR TITLE
provide working example and update README

### PR DESCRIPTION
**Pull request by afcondon**
**Description of the change**

The example referenced in the README didn't exist and the code snippet had some problems too (`liftEff`, unspecified imports etc) that would have made it quite difficult for, say, a total beginner to fix. 

I noticed after i'd done this that there is an issue #23 addressing this and that the `test` directory version is updated...however, it is not, in fact, a test and doesn't run with `spago test` so if this PR is accepted i would suggest either removing the test dir or at least raising an issue to make it actually test the library.

(it would also be fine to replace this example with the much simpler Cookbook recipe ReadPrintFileContentsNode or pointer to same)

### CODE REVIEW

File: example/packages.dhall [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          , "now"
+          ]
+      , repo =
+          "https://github.com/hdgarrood/purescript-benchotron.git"
+      , version =
+          "v7.0.0"
+      }
+  }
+-------------------------------
+-}
```
Comment (JordanMartinez) [inline example/packages.dhall:118]: Can you get rid of these comments in the `examples/packages.dhall` file?

**Comment by JordanMartinez** [comment]
Sorry I just realized this library is using `bower`. I don't think we should switch to `spago` in this library.

**Comment by afcondon** [comment]
> Sorry I just realized this library is using `bower`. I don't think we should switch to `spago` in this library.

maybe just PR a much simpler change simply correcting the read me and pointing to cookbook example, then?

**Comment by JordanMartinez** [comment]
Sounds fine to me, though I think another core member should comment on this, too.

**Comment by hdgarrood** [comment]
If we drop the `stringutils` dependency, then this example won't need any dependencies which aren't already dependencies of the library. So let's do that - we shouldn't be bringing in stringutils anyway - and then we can reuse the existing bower.json to ensure that the dependencies for the example are installed. We should also add an `example` script in the package.json so that `npm run example` does work; it should be along the lines of `pulp run --include example/ --main Example.Main`.

**Comment by afcondon** [comment]
I took a shot at getting it to work with `pulp` etc, but couldn't get it to compile, it's been so long since i used `bower/pulp` i probably messed something up, but i ran out of time to find it.

**Comment by hdgarrood** [comment]
The CI setup uses bower/pulp already, so you can use the same commands as we have there and it should work.

**Comment by afcondon** [comment]
Yeah, i expected:

```
npm install
bower install
npm run build
```

to be able to build it in a clean git clone, but Bower seems to be pulling in stuff leading to `Prim.Coerce not found` - is the CI all already going with `purs 0.14`?

**Comment by thomashoneyman** [comment]
@afcondon yes, the Node libraries are all expecting the 0.14 compiler and are using master / main branches for dependencies in prep for the 0.14 release.

If the files you have implemented compile on 0.13.8, perhaps you could just apply them to a fresh clone without building and rely on CI for verification? Then your PR could only include the README and Main.purs changes and we can properly review. This library wasn’t much affected by 0.14.

**Comment by JordanMartinez** [comment]
Closing as this repo is deprecated.

**Comment by JordanMartinez** [comment]
Closing as this repo is deprecated.

**Pull request closed without merging**